src/View.js

```diff
@@ -82,4 +109,10 @@
       }
     });
   }
+
+  LifeGamification.view.createSkill = function () {
```

Comment:
Does this function really creates a Skill? This sounds rather like a constructor to Skill class.